### DISCUSSION TITLE
Landscape/Tool Collection

### DISCUSSION THREAD

**Discussion by mkorbi** [cncf/tag-env-sustainability#22] [General]
Develop a list of tools that can impact, measure or positively influence the environmental sustainability footprint.
Potentially cluster them by categories, maybe give a description of how they impact.

A future outlook would be to PR a https://landscape.cncf.io/

**Comment by wojtekidd** [comment]
I think this should encompass both the tools that are a part of the CNCF and those that aren't which we might discover in due process.

**Reply by mkorbi** [reply]
+1

**Comment by rossf7** [comment]
Green Software Foundation have an [awesome-green-software](https://github.com/Green-Software-Foundation/awesome-green-software) list. This might be a useful starting point and I think there will be some overlap.

However I think a list related to CNCF projects would be very useful. GSF are also currently keeping their list focused on carbon / electricity so tools like autoscaling are not present. I know because I wanted to add KEDA ;)

**Reply by mkorbi** [reply]
@rossf7 just heads up there is some communications on that also in the mailing list https://lists.cncf.io/g/cncf-wg-env-sustainability/

**Comment by catblade** [comment]
Document started here:  https://github.com/catblade/wg-env-sustainability/tree/landscape_document/documentation/WhitePapers

If wanting to use hackmd.io, please message me on slack and I'll get you set to play.
